### PR TITLE
The Pumped Up Kicks Update

### PR DESCRIPTION
**Pull request by alsoandanswer**
Shoes. New shoes. Look at them. Ported (mostly) from /tg/. With some modifications by yours truly.

dreammaker_Fqwko20Fak

Holy shit just shoot me.

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

**Review by Alberyk — APPROVED** [review]

File: code/modules/client/preference_setup/loadout/loadout_shoes.dm [hunk 1]
```diff
@@ -96,16 +105,22 @@
 /datum/gear/shoes/boots/New()
 	..()
 	var/list/shoes = list()
-	shoes["black boots, short"] = /obj/item/clothing/shoes/jackboots
-	shoes["black boots, knee"] = /obj/item/clothing/shoes/jackboots/knee
-	shoes["black boots, thigh"] = /obj/item/clothing/shoes/jackboots/thigh
+	shoes["jackboots"] = /obj/item/clothing/shoes/jackboots
+	shoes["jackboots, cavalry"] = /obj/item/clothing/shoes/jackboots/thigh
```
Comment (alsoandanswer) [inline code/modules/client/preference_setup/loadout/loadout_shoes.dm:109]:
```suggestion
	shoes["jackboots, cavalry"] = /obj/item/clothing/shoes/jackboots/cavalry
```

File: code/modules/clothing/shoes/oxfords.dm [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+	icon_state = "oxford_brown"
+	item_state = "oxford_brown"
+
+/obj/item/clothing/shoes/laceup/colourable
+	name = "oxford shoes"
+	icon_state = "oxford_colour"
+	item_state = "oxford_colour"
+
+/obj/item/clothing/shoes/laceup/steeltoed
+	name = "steel-toed  shoes"
```
Comment (Generalcamo) [inline code/modules/clothing/shoes/oxfords.dm:26]:
```suggestion
	name = "steel-toed oxford shoes"
```
Weird space in the middle

**Review by Generalcamo — CHANGES_REQUESTED** [review]

**Comment by alsoandanswer** [comment]
!review

**Review by NonQueueingMatt — APPROVED** [review]

**Comment by Geevies** [comment]
Feedback thread please.

Many toeless boots aren't actually toeless.

Will put this in feedback thread also (as an opinion), but don't remove the short/knee/thigh boots. I quite like them. The shorter jackboots look like puddlestompers.

**Comment by NonQueueingMatt** [comment]
Cavalry boots are invisible

**Comment by alsoandanswer** [comment]
> Feedback thread please.
> 
> Many toeless boots aren't actually toeless.
> 
> Will put this in feedback thread also (as an opinion), but don't remove the short/knee/thigh boots. I quite like them. The shorter jackboots look like puddlestompers.

I'm only removing the thighboots, because they're the most egregious of the three, and is unused by most people -
I've asked around, and people only typically use the kneeboots instead.

> Cavalry boots are invisible

Fixed.

**Comment by Geevies** [comment]
> 
> I'm only removing the thighboots, because they're the most egregious of the three, and is unused by most people - I've asked around, and people only typically use the kneeboots instead.
> 

I'd love to tackle this on the forum instead of on github, but since there doesn't appear to be a thread going up, I'll do it here.

Asked around where, and who's "people"? Egregious how? I use it. I like it.

I'm fairly sure unused by most people isn't a valid excuse for removal, otherwise we'd be clearing out 99% of the loadout, such as the aurora miniature. At the very least the toeless variants don't seem to be resprited at all, as picture below, so there's no real reason to remove them other than... opinion, I guess.

(yes i know the fit is shit i just wanted a quick comparison of sprites on an unathi)
image

**Comment by Geevies** [comment]
Thank you! I appreciate getting the riding boots added.

One thing I noticed while playing- if I lie down my boots seem to get a different rotation applied to them, causing them to float off into the aether. I'm just not sure if it's related to this testmerge or something else alongside this one.

**Pull request merged**